docs/user-guide/src/about.md

```diff
@@ -0,0 +1,12 @@
+
+CeresDB is a timeseries database. However, CeresDB's goal is to handle both timeseries and analytic workloads compared with the traditional ones, which usually have a poor performance in handling analytic workloads.
```

Comment:
I write a simplified one:

```suggestion
CeresDB is a timeseries database that aims to handle both timeseries and analytic workloads efficiently.
```

Motivation:
- the _However_ is a bit abrupt
- to my understanding, traditional is bad at spreading tags, not every analytic scenario. So I think only to state we can handle both is enough as a short introduction.